stdlib/2/os/__init__.pyi

```diff
@@ -94,10 +95,58 @@ WUNTRACED = 0  # Unix only
 
 TMP_MAX = 0  # Undocumented, but used by tempfile
 _PathType = Union[bytes, Text]
-_StatVFS = NamedTuple('_StatVFS', [('f_bsize', int), ('f_frsize', int), ('f_blocks', int),
-                                   ('f_bfree', int), ('f_bavail', int), ('f_files', int),
-                                   ('f_ffree', int), ('f_favail', int), ('f_flag', int),
-                                   ('f_namemax', int)])
+
+class stat_result(NamedTuple('stat_result', [
+        ('st_mode', int),  # protection bits,
```

Comment:
I don't think we need all of those comments. Typeshed is for types, not documentation.